settings.php

```diff
@@ -108,11 +108,20 @@
             15, $jointimeselect);
     $settings->add($firstabletojoin);
 
-    $displayleadtime = new admin_setting_configcheckbox('zoom/displayleadtime',
-            get_string('displayleadtime', 'mod_zoom'),
-            get_string('displayleadtime_desc', 'mod_zoom'), 0, 1, 0);
-    $settings->add($displayleadtime);
-    $settings->hide_if('zoom/displayleadtime', 'zoom/firstabletojoin', 'eq', 0);
+    if ($CFG->branch >= 37) {
```

Comment:
$CFG->release seems to be the only machine-safe and human-readable way to check which Moodle version we're on. Moodle adds some extra characters, so they want you to pass it through `normalize_version()` first. Last, the appropriate PHP function for version comparisons is `version_compare()`.

```suggestion
    if (version_compare(normalize_version($CFG->release), '3.7.0', '>=')) {
```

Moodle 3.10 broke the unsafe assumption that the minor version would always be a single digit. The Moodle core code itself makes this assumption here: [convert_branch_numbering_format()](https://github.com/moodle/moodle/blob/6b804f9e1f8271adc9fe64a18fd318ca9537c674/lib/classes/update/api.php#L303-L312), [test_convert_branch_numbering_format()](https://github.com/moodle/moodle/blob/6b804f9e1f8271adc9fe64a18fd318ca9537c674/lib/tests/update_api_test.php#L47-L57)

@mudrd8mz I just noticed this issue with $CFG->branch "parsing" and tests, but I need to run so don't have time to open a tracker.